engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -2402,6 +2401,7 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
           LOG.warn("Cannot flush command history");
         }
       }
+      yarnClient.stop();
```

Comment:
It can be either stop() or close(), but it should be consistent. Should YarnAppLauncherImpl.close() also relay to stop()?